ddtrace/contrib/cassandra/session.py

```diff
@@ -253,7 +253,9 @@ def _sanitize_query(span, query):
         # reset query if a string is available
         resource = getattr(query, "query_string", query)
     elif t == 'BatchStatement':
-        resource = '; '.join(q[1] for q in query._statements_and_parameters[:2])
```

Comment:
Below is the line

```python
 span.resource = stringify(resource)[:RESOURCE_MAX_LENGTH]
```

We documented an issue with `stringify` failing to decode unicode (and I think all non-ascii encodings) in #537. 

Because we set the resource to the query, we were now calling `stringify` on the potentially non-ascii encoded query.

Reply:
ah, makes sense